examples/helloapache/README.md

```diff
@@ -1,8 +1,10 @@
+# helloapache
+
 This is an atomic application based on the nulecule specification. Kubernetes is currently the only supported provider. You'll need to run this from a workstation that has the atomic CLI and kubectl client that can connect to a kubernetes master.
 
-It's a single pod based on the centos/httpd image, but you can use your own.
+It's a single container application based on the centos/httpd image, but you can use your own.
 
```

Comment:
suggestion, s/It's a single container application/ It's a single container, single pod application/. 

I think the single pod information should be present too. As pod is the unit of containers for k8s.